src/network/mt_nonblocking/ServerImpl.cpp

```diff
@@ -130,6 +140,21 @@ void ServerImpl::Join() {
     for (auto &w : _workers) {
         w.Join();
     }
+    _workers.clear();
```

Comment:
12   Object "src/afina", at 0x560ac42726c1, in Afina::Network::MTnonblock::ServerImpl::~ServerImpl()
11   Object "src/afina", at 0x560ac4273502, in Afina::Network::MTnonblock::ServerImpl::Join()
10   Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6", at 0x7fc7d40fc70b, in std::thread ::join()
9    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6", at 0x7fc7d40c7c9c, in std::__throw_system_error(int)